pom.xml

```diff
@@ -27,7 +27,7 @@
 				<plugin>
 					<groupId>org.jboss.tools.tycho-plugins</groupId>
 					<artifactId>target-platform-utils</artifactId>
-					<version>0.21.0</version>
+					<version>0.22.1-SNAPSHOT</version>
```

Comment:
Related insofar as it was time to move to the latest versions of Tycho and platform-utils. Not strictly required for adding Mylyn Wikitext feature.

Not using a property because target-platforms don't use parent pom so can't use same value in there. 

You want these extracted into properties? Does that need to be in a separate PR?